app/templates/components/addon-tabs/panel.hbs

```diff
@@ -1,9 +1,12 @@
-<div
-  id="addon-panel-{{@tabId}}"
-  role="tabpanel"
-  tabindex="0"
-  aria-labelledby="addon-tab-{{@tabId}}"
-  class="addon-tabs--content {{if (eq @currentTab @tabId) "active-tab"}}"
->
-  {{yield}}
-</div>
\ No newline at end of file
+{{#let (eq @currentTab @tabId) as |isTabActive|}}
+  <div
+    id="addon-panel-{{@tabId}}"
+    role="tabpanel"
+    tabindex="0"
+    aria-labelledby="addon-tab-{{@tabId}}"
+    class="addon-tabs--content {{if isTabActive "active-tab"}}"
+    hidden={{not isTabActive}}
```

Comment:
The hidden attribute seemed to be missing in the previous implementation so I added that as well.

Reply:
The `not` helper isn't actually coming from the project. It only works because an addon depends on `ember-truth-helpers` already.

Reply:
Opened #589 for this.